.vib/gitlab-runner/goss/vars.yaml

```diff
@@ -0,0 +1,11 @@
+binaries:
+  - gitlab-runner
```

Comment:
You should also check for `gosu`, as the Dockerfile includes it:

https://github.com/bitnami/containers/blob/d31477da6b9e83c8b255284531ca9db2596a1562/bitnami/gitlab-runner/15/debian-11/Dockerfile#L25